content/en/blog/2025/otel-collector-k8s-discovery/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Kubernetes annotation based discovery for OpenTelemetry Collector
+linkTitle: Kubernetes annotation discovery
+date: 2025-01-20
+author: >
+  [Dmitrii Anoshin](https://github.com/dmitryax) (Cisco/Splunk), [Christos
+  Markou](https://github.com/ChrsMark) (Elastic)
+cSpell:ignore: Dmitrii Anoshin Markou
```

Comment:
```suggestion
sig: Collector
issue: opentelemetry-collector-contrib#34427
cSpell:ignore: Dmitrii Anoshin Markou
```

We ask SIG members as well to make sure that their blog posts are sponsored, I assume that the Collector SIG has agreed to publish this and is going to review, so I will not ask you to raise an additional issue, but please add these annotations (see https://opentelemetry.io/docs/contributing/blog/ for more details)